src/manifold/CMakeLists.txt

```diff
@@ -32,4 +32,17 @@ target_compile_features(${PROJECT_NAME}
 )
 
 install(TARGETS ${PROJECT_NAME} EXPORT manifoldTargets)
+
+if(MANIFOLD_HULL)
+add_library(hull module/hull.cpp)
+target_link_libraries(hull
+    PRIVATE manifold quickhull
```

Comment:
@pca006132 here's an attempt at making this even more modular, I believe going toward what @kintel was suggesting. This appears to work, though I'd love some feedback on best practices and such. I still need to fix up the bindings.